<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>member_widget</class>
 <widget class="QWidget" name="member_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>38</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>38</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>38</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#btn {border-radius: 6px;}
#btn:hover, #btn:focus, #btn[checked=&quot;true&quot;] {background-color: #303030;}
#icon_status {padding: 2px;}

#label_status {
	margin-left: 4px;
	font-size: 7pt;
	font-weight: 600;
}

#label_username {
	margin-left: 1px;
	font-size: 11pt;
	font-weight: 600;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="layout_widget">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="FrameButton" name="btn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::StrongFocus</enum>
     </property>
     <layout class="QGridLayout" name="layout_btn" rowstretch="1,0" columnstretch="0,0,1">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="IconWidget" name="icon_status">
        <property name="minimumSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../resources/index.qrc">:/status/online</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="MaskedImage" name="icon_avatar">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../resources/index.qrc">:/icons/avatar</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <layout class="QVBoxLayout" name="layout_labels" stretch="1,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string notr="true">Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>Online</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>icon_avatar</zorder>
     <zorder>icon_status</zorder>
     <zorder>verticalLayoutWidget</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameButton</class>
   <extends>QFrame</extends>
   <header>widgets.common</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QLabel</extends>
   <header>widgets.common</header>
  </customwidget>
  <customwidget>
   <class>MaskedImage</class>
   <extends>QLabel</extends>
   <header>widgets.common</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/index.qrc"/>
  <include location="../../../resources/index.qrc"/>
 </resources>
 <connections/>
</ui>
